examples/extjs_deftjs/.gitignore

```diff
@@ -4,3 +4,10 @@ coffee
 tools
 css/base.css
 css/bg.png
+
```

Comment:
Ah, you're right, it's a leftover from a previous change. But still nice to remove it though.